docs/source/user/runtime_view.rst

```diff
@@ -147,9 +147,17 @@ meanings):
 - critical - Something went wrong, but recovery is not possible
 
 Severity increases from "trace" to "critical" such that "trace" is the least
-important and "critical" is the most severe log statements. In practice, the
-value of a computed result usually falls under debug or trace. There's at least
-two ways to do that:
+important and "critical" is the most severe log statements. This means that
+all logging statements set as trace and below are enabled if the severity of
+the logger is set to trace. If the logger is set to debug, all debug, info,
+warn, error, and critical message logs are enabled, while trace message logs
+are not. For completeness, info includes info, warn, error, critical, and
+warn includes warn, error, critical, and error includes error and critical,
+and critical only includes critical. 
```

Comment:
I honestly think explicitly stating all of these is a redundant with the above explanation. I would only provide one explicit example of one of the intermediate values, probably either `info` or `warn`.